docsrcs/yui-ibc-solidity/docs/minitoken/contract.md

```diff
@@ -0,0 +1,318 @@
+---
+sidebar_position: 3
+---
+
+# Create a contract
+
+We will implement a token that can be transferred between two ledgers using IBC.
+
+There is a token standard called [ICS-20](https://github.com/cosmos/ibc/tree/master/spec/app/ics-020-fungible-token-transfer),
+but we will use a simple one here.
+
+While ICS-20 uses denomination to distinguish the source ledger,
+the MiniToken implemented here handles the issuer's ledger without distinction.
+
+## Basic Functions
+
+It has the following basic operational functions:
+
+- mint: issue a new token for a given account
```

Comment:
I think these function names may be easier to read if you enclose them in ``.
e.g. `mint`, `burn`